src/HIE/Bios/Cradle.hs

```diff
@@ -495,6 +513,8 @@ stackCradle wdir mc =
     , cradleOptsProg   = CradleAction
         { actionName = Types.Stack
         , runCradle = stackAction wdir mc
+        , getGhcPath = optional $ fmap trim $
+            readProcess "stack" ["exec", "which", "--silent", "--", "ghc"] ""
```

Comment:
This may not be portable neither:
```
PS D:\dev\ws\haskell\hls> stack exec which --silent -- ghc
/d/bin/stack/x86_64-windows/ghc-8.6.5/bin/ghc
```
stack uses `which` from its msys installation but the path has the msys format, not sure if it could be usable as is.

What about

```
D:\dev\ws\haskell\hls>stack path --compiler-exe
D:\bin\stack\x86_64-windows\ghc-8.6.5\bin\ghc-8.6.5.exe
```